mypy/semanal_enum.py

```diff
@@ -107,7 +107,8 @@ def parse_enum_call_args(self, call: CallExpr,
         if len(args) < 2:
             return self.fail_enum_call_arg("Too few arguments for %s()" % class_name, call)
         if len(args) > 2:
-            return self.fail_enum_call_arg("Too many arguments for %s()" % class_name, call)
+            if call.arg_kinds != [ARG_POS, ARG_POS, ARG_POS]:
```

Comment:
This doesn't make much sense in the light of the next `if` condition: if this one passes the next one (on line 112) will still fail.

Reply:
@JelleZijlstra Thanks for your review. Could you tell me what does `ARG_POS` mean?